### DISCUSSION TITLE
How to get the debian cloudImage working

### DISCUSSION THREAD

**Discussion by nano2007** [kairos-io/kairos#954] [Q&A]
I am trying to follow the documentation to create a debian based cloud image. The yaml file for creating the osartifact manifest is:

[kairos-debian.yaml.txt](https://github.com/kairos-io/kairos/files/10821089/kairos-debian.yaml.txt)

I then create a VM in a KVM box by importing the created raw disk image. Here are the boot logs:

[kairos-debian-boot.log](https://github.com/kairos-io/kairos/files/10821077/kairos-debian-boot.log)

After the VM boots, I do not see an installation or reboot happening.

The same config file works for the opensuse:latest image (where I can see the installation and reboot and the COS_STATE and COS_PERSISTENT partitions exist after the reboot).

Any help is greatly appreciated. Thank you !

**Comment by mudler** [comment]
Hey @nano2007 :wave: welcome!

Can you file up an issue? this looks like a bug with our debian flavor

**Reply by nano2007** [reply]
converted this to a bug: https://github.com/kairos-io/kairos/issues/955
